app/commands/training_data/code_tags_sample/update_tags.rb

```diff
@@ -0,0 +1,31 @@
+class TrainingData::CodeTagsSample::UpdateTags
+  include Mandate
+
+  initialize_with :sample, :tags, :user
+
+  def call
+    ActiveRecord::Base.transaction do
+      sample.lock!
```

Comment:
I have a small preference for:
```suggestion
    sample.with_lock do
```

Which does the same thing.